modules/installation-requirements-user-infra.adoc

```diff
@@ -48,7 +48,7 @@ endif::ibm-z[]
 The bootstrap, control plane, and compute machines must use the {op-system-first} as the
 operating system.
 
-Note that {op-system} is based on Red Hat Enterprise Linux 8 and inherits all of its hardware certifications and requirements.
+Note that {op-system} is based on {op-system-base-full} and inherits all of its hardware certifications and requirements.
```

Comment:
@vrutkovs 
s/{op-system-base-full}/{op-system-base-full} 8

I think we need the RHEL version here.

Reply:
Good catch. Added `8` there and hid it in OKD docs - FCOS is a rolling release and may switch to versions arbitrarily

Reply:
@vrutkovs I merged this PR but I'm not seeing the `8` in master. 
https://github.com/openshift/openshift-docs/blob/master/modules/installation-requirements-user-infra.adoc

Reply:
Added it in https://github.com/openshift/openshift-docs/pull/21640